addons/base_vat_autocomplete/views/res_company_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+  The purpose of this view is to move the VAT field at the top of the 
+  company form. That way, the user will fill it first
+-->
+<odoo>
+    <record id="company_form_vat_autocomplete" model="ir.ui.view">
```

Comment:
NamingStuff: we try to stick on <model>_view_<view_type>(_<suffix>) for naming xml ids. In that case it would be something like res_company_view_form_inherit_vat

Reply:
Up ! New convention is just res_company_view_form for simplification.